src/scripts/main.js

```diff
@@ -1,3 +1,251 @@
+/* eslint-disable max-len */
 'use strict';
 
-// write code here
+const QUERY = {
+  'Name': 0,
+  'Position': 1,
+  'Office': 2,
+  'Age': 3,
+  'Salary': 4,
+};
+
+const body = document.querySelector('body');
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+const employees = tbody.children;
+
+let desc = false;
+let query = '';
+
+function sortEmployees(sortedQuery) {
+  if (sortedQuery < 3) {
```

Comment:
3 is a magic number
Consider will be better if you create a constants for magic number and use it everywhere